application-collabora-ui/src/main/resources/Collabora/Code/UI.xml

```diff
@@ -232,6 +232,12 @@ require(['jquery', 'xwiki-l10n!collabora-attachment'],function($, l10n) {
   $(document).on('xwiki:html5upload:fileFinished', decorateWithCollaboraButtons);
   $(document).on('xwiki:docextra:loaded', decorateWithCollaboraButtons);
 
+  /**
+  * Collabora event that other developers can trigger to initialize the process of decorating the buttons of their
+  * livedata
```

Comment:
```suggestion
  * Custom event fired for initializing the process of adding Collabora buttons. This can be helpful for UI 
  * elements that were loaded later and require these buttons.
```